defaults/main.yml

```diff
@@ -1 +1,18 @@
-# defaults file for localectl
+# "become" handling
+localectl_become: true
```

Comment:
Is `localectl_become_user` not needed for this role?

Reply:
You can become any user you want, as long as that user is `root`.